clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/model/AmazonLoadBalancer.groovy

```diff
@@ -57,6 +59,6 @@ class AmazonLoadBalancer implements LoadBalancer {
 
   @Override
   int hashCode() {
-    getAccount().hashCode() + getName().hashCode() + getType().hashCode() + getServerGroups().hashCode() + getRegion().hashCode()
+    getAccount().hashCode() + getName().hashCode() + getType().hashCode() + getServerGroups().hashCode() + getRegion().hashCode() + getTargetGroups().hashCode()
```

Comment:
not sure who did the initial hashCode on here but it is kind of bad - probably worth letting the IDE auto-generate one